src/main/webapp/css/style.css

```diff
@@ -23,6 +18,12 @@ h1 {
   color: #0F9D58;
 }
 
+.searchbox {
+  width: 60%;
+  min-width: 400px;
+  max-width: 1000px;
```

Comment:
1000px is actually really large. For comparison, the google.com searchbox ranges from 400px to 584px, and that's including the search and microphone buttons inside.

I'm actually not sure about the 60% anymore. Why waste 40% of the screen width on small screens?